meta-mender-core/classes/mender-setup-image.inc

```diff
@@ -20,18 +20,18 @@ MENDER_BOOT_PART_MOUNT_LOCATION_mender-image-uefi = "/boot/efi"
 # Update fstab for Mender
 ROOTFS_POSTPROCESS_COMMAND_append += " mender_update_fstab_file ; "
 mender_update_fstab_file() {
-    if ! [ "${@bb.utils.contains('DISTRO_FEATURES', 'mender-image-ubi', 'true', 'false', d)}" ]; then
+    if ! ${@bb.utils.contains('DISTRO_FEATURES', 'mender-image-ubi', 'true', 'false', d)}; then
```

Comment:
Actually the reason I wrote it like that is that I don't understand how this could ever have worked, but apparently it did. However it didn't work once I introduced `mender-image-bios`, so I had to change it.

So basically I don't fully understand it either, other than the fact that the new way clearly is better.

Reply:
In fact, I think I'll write that in the commit message.. :-)